docker-compose.yml

```diff
@@ -20,7 +20,7 @@ services:
       - rabbitmq_data:/var/lib/rabbitmq
 
   django:
-    image: geonode/django
+    image: geonode/geonode
```

Comment:
This change requires probably that we modify the name into our docker hub. What is the mean of such commit?

Reply:
I revert image name to geonode/django